src/screenshot/snippingtool.py

```diff
@@ -0,0 +1,17 @@
+from subprocess import call
+from PIL import ImageGrab
+from libraries.pyrobot import Robot
+
+_r = Robot()
+
+
+def take_screenshot_whole(path):
+    image = _r.take_screenshot()
+    image.save(path, "PNG")
```

Comment:
Please replace "PNG" with "png". Even though Windows makes everything case insensitive, we should do this for consistency.